.gitignore

```diff
@@ -1,18 +1,109 @@
-# Compiled python modules.
-*.pyc
+# Byte-compiled / optimized / DLL files
```

Comment:
This is a very user-specific ignore list, more suitable for `git config --global core.excludesfile $HOME/.gitignore` than versioning in this repo.

Reply:
Thank you!